src/test/java/htsjdk/samtools/CSIIndexTest.java

```diff
@@ -29,8 +29,8 @@
     private static Bin bin9 = new Bin(1, 135853);
     private static Bin bin10 = new Bin(1, 163);
 
-    @BeforeTest
-    public void init() {
+    @BeforeClass
+    public static void init() {
```

Comment:
I'm confused, what is the benefit of using BeforeClass here, but BeforeTest for IntervalTreeTest.

Reply:
I used `BeforeClass` since the initialization needed to be static. However, looking at this class again, all test methods seems to be static, which is unusual, so I've made everything non-static and reverted back to `BeforeTest`.